pkg/api/config_test.go

```diff
@@ -669,13 +669,62 @@ func TestValidateTestSteps(t *testing.T) {
 		errs: []error{
 			errors.New("test[1].ref: duplicated name \"as\""),
 		},
+	}, {
+		name: "Test step with forbidden parameter",
+
+		steps: []TestStep{{
+			LiteralTestStep: &LiteralTestStep{
+				As:                "as",
+				From:              "from",
+				Commands:          "commands",
+				Resources:         resources,
+				OptionalOnSuccess: func(b bool) *bool { return &b }(true)},
```

Comment:
This is neat but generally I think we just do something like the following at the top of the `func Test...`

```go
yes := true

// ...

    OptionalOnSuccess: &yes
```

Reply:
Fixed